docker/demo-cluster.sh

```diff
@@ -18,20 +18,15 @@
 #
 
 
-PROJECT_VERSION=latest
+AMORO_TAG=master-snapshot
```

Comment:
Does this mean we will usually update the image with the tag "master-snapshot"?

Reply:
Yes,  I will configure a github ci, once PR merge to master, a new image will be build and push.